assets/js/components/Message/index.js

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+
+const Message = (props) => (
+    <div>
+        <p>
+            {props.message} please select a spot on the board!
```

Comment:
The spacing seems pretty intense here (like too many spaces)

Reply:
The tabs?

Reply:
Yeah, 2 spaces is standard for me.